pyproject.toml

```diff
@@ -49,3 +49,28 @@ unfixable = ["F401"]
 [tool.ruff.mccabe]
 # Unlike Flake8, default to a complexity level of 10.
 max-complexity = 10
+
+[tool.mypy]
```

Comment:
Do you think we need to include mypy configs? This will raise a lot of warnings, right?

Reply:
Only if you explicitly run mypy, otherwise no. Mypy is only configured, not included in pre-commit or workflows. This is a pretty common configuration for production level code. I plan to fix other files as well in future PR, there are some errors which cause confusion.